build/source/driver/multi_driver.f90

```diff
@@ -589,15 +628,15 @@ program multi_driver
   ! (i.e., where multiple HRUs drain to a basin-average aquifer)
   case(singleBasin)
    bvarStruct%gru(iGRU)%var(iLookBVAR%basin__AquiferStorage)%dat(1) = 1._dp
-   do iHRU=1,gru_struc(iGRU)%hruCount
-    progStruct%gru(iGRU)%hru(iHRU)%var(iLookPROG%scalarAquiferStorage)%dat(1) = 0._dp  ! set to zero to be clear that there is no local aquifer storage in this configuration
+   do localHRU=1,gru_struc(iGRU)%hruCount
+    progStruct%gru(iGRU)%hru(localHRU)%var(iLookPROG%scalarAquiferStorage)%dat(1) = 0._dp  ! set to zero to be clear that there is no local aquifer storage in this configuration
    end do
   case default; call handle_err(20,'unable to identify decision for regional representation of groundwater')
  end select
 
  ! initialize time step length for each HRU
- do iHRU=1,nHRU
-  dt_init(iGRU)%hru(iHRU) = progStruct%gru(iGRU)%hru(iHRU)%var(iLookPROG%dt_init)%dat(1) ! seconds
+ do localHRU=1,gru_struc(iGRU)%hruCount
```

Comment:
nHRU was the total number of HRUs for the current GRU, so I think that should be ok.